src/components/ModalLedgerResetTokenSignatures.js

```diff
@@ -0,0 +1,168 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, {useState, useEffect} from 'react';
+import $ from 'jquery';
+import ledger from '../utils/ledger';
+import tokens from '../utils/tokens';
+import { IPC_RENDERER } from '../constants';
+import { t } from 'ttag';
+
+/**
+ * Component that shows a modal to add one specific unknown token to the wallet
+ *
+ * @memberof Components
+ */
+function ModalLedgerResetTokenSignatures() {
+  const modalId = "resetTokenSignatures";
+  const [onLedger, blockClose] = useState(false);
```

Comment:
Should we rename this like in the other PR?

```
const [waitingLedger, setWaitingLedget] = useState(false);
```

Reply:
Done